src/reports/components/report-sections/urls-summary-section.scss

```diff
@@ -13,6 +13,9 @@
     h2 {
         margin-bottom: 16px !important;
     }
+    .total-urls {
+        font-weight: 600;
```

Comment:
Can this use `$fontWeightSemiBold` from `fonts.scss` instead of hard-coding the number?